src/types/BaseField.ts

```diff
@@ -11,12 +11,18 @@ export enum BaseFieldDataType {
 	BOOLEAN = 'boolean',
 }
 
+export enum BaseFieldScope {
```

Comment:
It would be handy if there were a way to define the enum once and read it in the other place.